HUD/AddBlipForRadius.md

```diff
@@ -10,9 +10,12 @@ Blip ADD_BLIP_FOR_RADIUS(float posX, float posY, float posZ, float radius);
 
 
 ## Parameters
-* **posX**: 
-* **posY**: 
-* **posZ**: 
-* **radius**: 
+* **posX**: World coordinate X of the blip's center.
+* **posY**: World coordinate Y of the blip's center.
+* **posZ**: World coordinate Z of the blip's center.
+* **radius**: Radius of the blip.
 
 ## Return value
+Returns a handle for the blip created.
+
+Create a circle blip on the minimap and pause menu map. You need to set the colour and alpha of the blip using the handle created.
```

Comment:
Please move this line above the Parameters section for it to appear in the description. Right now it's part of the Return value description.